content/en/docs/reference/config/analysis/ist0158/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: PodsIstioProxyImageMismatchInNamespace
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+This message is similar to [IST0105](/docs/reference/config/analysis/ist0105/), which is a warning message for a single pod.
+
+This message occurs when the namespace is enabled for automatic sidecar injection, but some pods in the namespace are not having the correct sidecar injected.
+
+There are several possible causes for the pod:
+
+* Automatic sidecar injection is enabled (default enabled unless explicitly
```

Comment:
not sure how this is a cause for the warning message?

Reply:
Will revise to make situations more clear